### PR TITLE
Completed Random Contributor UI

### LINKED ISSUE #191: [Feature Request] Please enhance the UI of our Random Contributor Component
### Description

This is a longer design UI enhancement project. 

Please enhance the UI of the show button that then displays a random contributor's name and clickable image to their GitHub account. 

The idea here would be to rearrange this in regards to where it is displayed on the website and place it below the header component and above the [MasonryLayout Component](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/src/Components/MasonryLayout/MasonryLayout.jsx). The idea would be to have the text `Thanking Our Before I Die Contributors` and then showing a slick and professional UI card display of our contributors GitHub avatar and set it to use the random shuffle display algorithm that is seen when viewing the [RandomContributors Component]( https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/src/Components/randomcontributor/RandomContributors.jsx) so it randomly acts as a slide show cycling every 5 seconds as an example. 

Please see below this image of the potential idea. I've included it for you.I've included for you

### Screenshots

Here is where the newly designed [RandomContributors Component]( https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/src/Components/randomcontributor/RandomContributors.jsx) needs to go

Screenshot 2023-10-12 at 16 43 54

Here is a potential rough sketch of how the RandomContriubtor could be presented. Note that please don't follow the font or exact designing. This is to get you thinking of how to present this component to highlight our contributors and should blend well with the rest of the site this far.
Before I Die Wall (1280 x 720 px) (2)

This section is being pushed up from the issue #190. And if it still needs to be assigned, you can work on both. 
Screenshot 2023-10-12 at 16 46 01

### Checklist

- [X] I have checked for duplicate [issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues)
- [X] I have read the [Contribution Guidelines]((https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md))
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by LynxSumit**
Completed Random Contributor Component.

### CODE REVIEW

File: .hintrc
Comment (XanderRubio) [inline .hintrc:1]: I researched the purpose of the file and found out that it is used as a configuration file for the webhint tool, which is commonly used in React projects. This tool is designed to enhance website quality by verifying your code for best practices, performance, compatibility, and more. The .hintrc file is utilized to define the configuration for webhint, outlining what to test for and the severity it should have. This is a great way to enforce coding standards and best practices, as well as to identify potential issues within your codebase. Let me know if I missed anything as to why his file was created and in the mean time it makes sense to add to the project!

File: src/Components/randomcontributor/RandomContributors.jsx [hunk 1]
```diff
@@ -14,31 +13,27 @@ const RandomContributors = () => {
...
-      setTimeout(() => setCurrentIndex(newIndex), 750); // Pick new contributor after 0.75 seconds
-    } else {
-      setIsShown(true); 
-    }
-  };
+ useEffect(() => {
+  
+  setTimeout(() => {
+    setCurrentIndex(pickRandomContributor())
+  }, 5000);
```
Comment (XanderRubio) [inline src/Components/randomcontributor/RandomContributors.jsx:20]: Excellent 5 seconds!

Comment (LynxSumit) [reply src/Components/randomcontributor/RandomContributors.jsx:20]: thankss

File: .hintrc
Comment (LynxSumit) [reply .hintrc:1]: i was confused about this file .hintrc thanks for letting me know 👍👍

**Review by XanderRubio — APPROVED** [review]
Great work @LynxSumit! I appreciate you taking the time to improve the RandomContributor function and its location. Your assistance in developing the Before I Die code is greatly appreciated. I have tested the component on the preview link and it runs smoothly. Therefore, I am proceeding with the merging process now. Thank you!

**Comment by XanderRubio** [comment]
Congratulations @LynxSumit on having your pull requests merged! If you haven't already, please star this repository and share what you want to do before you die on issue #188. This way, you will receive an invitation for membership to the organization and get your name added to the contributors' list. Thank you for your contribution, have a great day!

**Comment by XanderRubio** [comment]
This merge successfully closes issue #191

**Pull request merged**